src/compiler/sys/dependencies.json

```diff
@@ -64,6 +64,7 @@
         "compiler/lib.es2022.full.d.ts",
         "compiler/lib.es2022.intl.d.ts",
         "compiler/lib.es2022.object.d.ts",
+        "compiler/lib.es2022.sharedmemory.d.ts",
```

Comment:
Nothing to change, just curious what this does/controls

Reply:
This JSON file gets built during [the build process](https://github.com/ionic-team/stencil/blob/4abaa0d8e065e565bdec9395fe1e6938e997d095/scripts/build.ts#L57) of building the compiler itself, and is "used" in a few places.
1. [When we invoke Stencil (any task)](https://github.com/ionic-team/stencil/blob/4abaa0d8e065e565bdec9395fe1e6938e997d095/src/cli/run.ts#L56). However, in a Node-only world, this doesn't appear to actually get used
2. [It's publicly exposed on `compiler/index.ts`](https://github.com/ionic-team/stencil/blob/4abaa0d8e065e565bdec9395fe1e6938e997d095/src/compiler/index.ts#L19),